charts/oes/charts/spinnaker/config/orca-local.yml

```diff
@@ -73,8 +52,10 @@ job:
                         value: "${{execution.startTime} .toString()}"
                       - name: user
                         value: "${execution.authentication.user}"
-                    image: 'opsmxdev/apline-with-curl'
+                    image: 'opsmxdev/policy:v1'
```

Comment:
Clarification: So, we put the script into the image now?

Reply:
yes, Abhi in order to reduce disturbance in Orca-local.yml